nautobot_ssot_infoblox/diffsync/client.py

```diff
@@ -1016,10 +1013,15 @@ def get_vlans(self):
         ]
         """
         url_path = "vlan"
-        params = {"_return_fields": "assigned_to,id,name,comment,contact,department,description,parent,reserved,status"}
+        params = {
+            "_return_as_object": 1,
+            "_paging": 1,
+            "_max_results": 100000000,
+            "_return_fields": "assigned_to,id,name,comment,contact,department,description,parent,reserved,status",
+        }
         response = self._request("GET", url_path, params=params)
         logger.info(response.json())
-        return response.json()
+        return response.json()["result"]
```

Comment:
This is because the changes required to get all of the VLANs in one call. This required adding paging, max_results, and the return_as_object params. Once those were added the response is different and has it included in a dict with results as the key and the list of VLANs as value. To match the other methods where it's returning just the list of results I included the results key here.